kernel/kernel-impl/src/main/java/org/sakaiproject/db/impl/BasicSqlService.java

```diff
@@ -2235,6 +2235,11 @@ else if (fields[i] instanceof Boolean)
 					pstmt.setBoolean(pos, ((Boolean) fields[i]).booleanValue());
 					pos++;
 				}
+				else if (fields[i] instanceof String)
```

Comment:
Could be combined in the instanceof check above.

Reply:
I don't understand which one? The only other one I see involving string is a null check.